src/styleguide/header-light.html

```diff
@@ -0,0 +1,101 @@
+<!doctype html>
```

Comment:
Yes it would: hence my comment in the PR decription actually. Can do that work, hadn't worked it through fully.

That HTML is generated by copy and paste and format from the browser currently. Manual, though based on reality.